src/main/mainWindow/index.js

```diff
@@ -85,6 +85,11 @@ async function createMainWindow() {
 	if (process.env.NODE_ENV === 'development') {
 		mainWindow.webContents.openDevTools();
 	}
+
+	if (app.commandLine.hasSwitch('start-hidden')) {
```

Comment:
AFAIK `app.commandLine` refers to flags passed to the Chromium processes, not being the same as `process.argv`.